lib/copy_color_spinor_offset.cu

```diff
@@ -22,27 +22,63 @@ namespace quda
     }
```

Comment:
It looks like you've clang formatted this entire file as opposed to the diff.  Can you revert this?  Else we end up with a lot of conflicts with other ongoing PRs.

Reply:
Done.